src/components/events/EventList.vue

```diff
@@ -45,20 +40,54 @@
 </template>
 
 <script>
+import { mapState, mapActions } from 'vuex';
+import TableHead from '@/components/_ui/TableHead.vue';
+import TableBodyNoData from '@/components/_ui/TableBodyNoData.vue';
+
 export default {
+  name: 'EventList',
+  components: { TableHead, TableBodyNoData },
   data() {
     return {
-      items: [],
+      title: 'Events',
+      tableHeadLinks: [
+        {
+          name: 'Add',
+          routeName: 'EventCreate',
+          class: 'btn-primary',
+          icon: 'fa-plus',
+        },
+      ],
     };
   },
-  // created() {
-  //   this.items = store.events;
-  // },
+  created() {
+    this.setCurrentHeader('...');
+  },
   mounted() {
-    this.$store.dispatch('application/setHeaderInfo', {
-      title: 'Events',
-      subtitle: 'Total: 0',
-    });
+    (async () => {
+      try {
+        await this.fetchItems();
+        this.setCurrentHeader(`Total: ${this.itemsTotal}`);
+      } catch (error) {
+        console.log('[Exception-EventList]', error);
+      }
+    })();
+  },
```

Comment:
Use async mounted instead of IIFE.

Reply:
Theoretically 'mounted' hook was designed to be synchronous, not asynchronous, but I can change this in the next commit.